index.html

```diff
@@ -7065,76 +7063,137 @@ <h3>Webhook Event Example</h3>
               which must be submitted through the <code>subscribeevent</code> form.
               The read-only parameter <code>subscriptionID</code> is returned by the subscription.
               The <i>WebhookThing</i> will then periodically POST to this callback URI with a payload defined by <code>data</code>.
```

Comment:
I think this language needs a bit of clarifications.
It is obvious that the data is *contained* in the event message, but it could be packaged into an envelop that contains metadata, such as timestamp, originator etc.

Please see https://github.com/w3c/wot-profile/pull/150 for details.

For clarity and better rendering in a printed/PDF version of the document, I suggest to split the example into two examples.

Reply:
> For clarity and better rendering in a printed/PDF version of the document, I suggest to split the example into two examples.

This is not needed anymore since #1279 provides a setup that allows to print all examples behind the tabs.